src/Apps/Order2/Routes/Details/Components/Order2DetailsMessage.tsx

```diff
@@ -82,16 +82,15 @@ const getMessageContent = (order): React.ReactNode => {
         <>
           <Text variant="sm">
             Thank you! Your offer has been submitted. You will receive an email
-            shortly with all the details. Please note making an offer doesn’t
+            shortly with all the details. Please note making an offer doesn't
```

Comment:
I remember we generally want apostrophe to be the curly one, rather than straight.